source/_docs/configuration/templating.markdown

```diff
@@ -175,6 +176,13 @@ Other state examples:
 {% if states('sensor.train_departure_time') in ("unavailable", "unknown") %}
   {{ ... }}
 
+{% if has_value('sensor.train_departure_time') %}
+  {{ ... }}
+
+{% if states.sensor.train_departure_time is has_value %}
```

Comment:
`states` should not be used in this context, as per our own documentation:

![image](https://user-images.githubusercontent.com/195327/225034110-220a29ae-4d35-4395-9ebc-c94572e1854a.png)